Cargo.toml

```
@@ -473,6 +473,9 @@ track_location = ["bevy_internal/track_location"]
 # Enable function reflection
 reflect_functions = ["bevy_internal/reflect_functions"]
 
+# Enable documentation reflection
+reflect_documentation = ["bevy_internal/reflect_documentation"]
```

feels a bit weird to document "documentation reflection" and then have it called `reflect_documentation`
```suggestion
documentation_reflection = ["bevy_internal/documentation_reflection"]
```

---

True, although you could say the same about `reflect_functions`. 
Would it make more sense to change the comment to match the flags?

---

most features are a noun, `reflect_documentation` is an action... I would prefer to rename `reflect_function`

---

well https://github.com/bevyengine/bevy/pull/14174#discussion_r1667259154
@alice-i-cecile an opinion there? should reflect be different than everything else for the sake of reflect features being grouped?

---

Yeah, I think it's worth it for grouping.